Header                                          header
infrastructure_camera_tracker/TrackedFeature[]  features
geometry_msgs/Point[]                           velocity  # estimated velocity (mov. avg.)
geometry_msgs/Point[]                           arrival_point # the arrival point estimate
float64[]                                       arrival_time  # the arrival time estimate (time to arrival)